ansible/README.md

```diff
@@ -280,6 +280,17 @@ This is usually not necessary, however in case you want to uninstall all prereqs
 ansible-playbook -i environments/<environment> prereq.yml -e mode=clean
 ```
 
+### Lean Setup
+To have a lean setup (no Kafka, Zookeeper and no Invokers as separate entities):
+
+At [Deploying Using CouchDB](ansible/README.md#deploying-using-cloudant) step, replace:
+```
+ansible-playbook -i environments/<environment> openwhisk.yml
```

Comment:
if you succeed in removing the lean playbook, then we can just a model option here `-e mode=lean` on the deploy controller playbook.